docs/openapi.yaml

```diff
@@ -1369,10 +1369,11 @@ components:
       properties:
         occurred:
           type: string
+          format: date-time
           description: |-
-            Date and time of the event, on the format
-            `2022-09-05T14:25:55Z`.
-          example: 2022-09-05T14:25:55Z
+            Date and time of the event, as timestamp on the format `2022-09-05T14:25:55Z`,
+            with or without milliseconds.
```

Comment:
We currently don't store the milliseconds on timestamps in our database so we can not send milliseconds

Reply:
Then we are future proof! 🎉